src/service/command/dice.ts

```diff
@@ -0,0 +1,81 @@
+import type {
+  CommandMessage,
+  CommandResponder,
+  HelpInfo
+} from './command-message.js';
+
+/**
+ * ダイスの管理者。面白いものの味方。
+ * 今後詳細値がほしい場合があるかも知れないので、各ダイスについてどんな出目が出たかを判断する。
+ */
+export interface DiceQueen {
+  /**
+   *
+   * @param {number} faces
+   * @param {number} HowManyRoll
+   * @return {Array<number>}
+   */
+  roll(faces: number, HowManyRoll: number): Array<number>;
```

Comment:
```suggestion
  roll(faces: number, howManyRolls: number): Array<number>;
```
命名規則は camelCase にしてほしいです